src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -173,28 +173,27 @@ def show_container_settings(resource_group_name, name, slot=None):
 def _filter_for_container_settings(settings):
     return {x: settings[x] for x in settings if x in CONTAINER_APPSETTING_NAMES}
 
-def add_hostname(resource_group_name, webapp, name, slot=None):
+def add_hostname(resource_group_name, webappName, name, slot=None):
     client = web_client_factory()
-    webapp = client.sites.get_site(resource_group_name, webapp)
-    binding = HostNameBinding(webapp.location, host_name_binding_name=name, site_name=webapp)
+    webapp = client.sites.get_site(resource_group_name, webappName)
+    binding = HostNameBinding(webapp.location, host_name_binding_name=name, site_name=webapp.name)
```

Comment:
Good, good, will do.  I know the other methods were unaffected, but I wanted to keep them consistent.  I will update today.